Author: Sven Joachim <svenjoac@gmx.de>
Description: Debian changes to xterm* terminfo entries
 The "rep" feature has been removed from xterm-new and derived terminfo
 descriptions, as it causes compatibility problems with older terminal
 emulators.
 .
 The "dec+sl" feature has been removed from xterm-new and derived terminfo
 descriptions, as that breaks updating the window title in several
 applications and caused display problems in mutt.
 .
 The xterm-debian terminfo entry is provided as an alias for xterm, as
 some ancient Debian systems used xterm-debian as $TERM.
Bug-Debian: https://bugs.debian.org/933053
Bug-Debian: https://bugs.debian.org/1010270
Bug-Debian: https://bugs.debian.org/1010278
Forwarded: not-needed
Last-Update: 2022-05-04

---
 misc/terminfo.src |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/misc/terminfo.src
+++ b/misc/terminfo.src
@@ -4881,12 +4881,12 @@ xterm+nofkeys|building block for xterm f
 xterm-p370|xterm patch #370,
 	npc,
 	kcbt=\E[Z, kent=\EOM, nel=\EE, use=ecma+index,
-	use=ansi+rep, use=ecma+strikeout, use=xterm+pcfkeys,
+	use=ecma+strikeout, use=xterm+pcfkeys,
 	use=xterm+nofkeys,
 
 # This version reflects the current xterm features.
 xterm-new|modern xterm terminal emulator,
-	use=dec+sl, use=xterm-p370,
+	use=xterm-p370,
 
 # This fragment is for people who cannot agree on what the backspace key
 # should send.
@@ -5549,7 +5549,7 @@ xterm-24|vs100|xterms|xterm terminal emu
 	lines#24, use=xterm-old,
 
 # This is xterm for ncurses.
-xterm|xterm terminal emulator (X Window System),
+xterm|xterm-debian|xterm terminal emulator (X Window System),
 	use=xterm-new,
 
 # This entry assumes that xterm's handling of VT100 SI/SO is disabled by
